src/elements/Content/Content.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+
+function Content(props) {
+  const {
+    children, className, ...rest,
+    } = props
+
+  const classes = cx(
+    'content',
+    className
+  )
+
+  return (
+    <div className={classes} {...rest}>{children}</div>
+  )
+}
+
+Content._meta = {
+  name: 'Content',
+  type: META.type.element,
+}
+
+Content.propTypes = {
+  /** Primary content of the List */
```

Comment:
I realize my comment says List while it should say Content. I've noticed the typos in the Content and Description files and will fix.